modules/apps/frontend-taglib/frontend-taglib-clay/bnd.bnd

```diff
@@ -47,7 +47,8 @@ Liferay-JS-Submodules-Bridge:\
 	clay-select,\
 	clay-sticker,\
 	clay-table,\
-	clay-tooltip
+	clay-tooltip,\
+	frontend-taglib-clay
```

Comment:
This is more of a question for my own lack of knowledge, but how some you need to specify this here? At first glance it seems odd to have to explicitly add this here in the module of the same name. But I also don't really know how `Liferay-JS-Submodules-Bridge` works.

Reply:
This will generate alias so that you can refer a component using it rather than its fully qualified name. It's a way to allow usage from a JSP like `frontend-taglib-clay/something` in our old bundler2-based architecture although it's always been a discouraged practice.